base/linalg/symmetric.jl

```diff
@@ -41,6 +41,15 @@ julia> Slower = Symmetric(A, :L)
 Note that `Supper` will not be equal to `Slower` unless `A` is itself symmetric (e.g. if `A == A.'`).
 """
 Symmetric(A::AbstractMatrix, uplo::Symbol=:U) = (checksquare(A); Symmetric{eltype(A),typeof(A)}(A, char_uplo(uplo)))
+Symmetric(A::Symmetric) = A
```

Comment:
tiny bit inconsistent that here the default uplo matches that of the input, whereas on all other types it defaults to upper